letsencrypt-nginx/letsencrypt_nginx/configurator.py

```diff
@@ -217,6 +218,38 @@ def choose_vhost(self, target_name):
 
         return vhost
 
+    def _create_vhost_conf_at_default_dir(self, target_name):
+        # Creates a new config file based on the location of the vhost conf files
+
+        # finds the most frequent path for vhost configurations
+        root_config = self.parser.loc["root"]
+        paths_freq = dict()
+        for vhost in self.parser.get_vhosts():
+            # ignore vhosts defined in the root config file to avoid generating
+            # files at the root config dir that are never included.
+            if vhost.filep == root_config:
+                continue
+
+            path = os.path.dirname(vhost.filep)
+            paths_freq.setdefault(path, 0)
+            paths_freq[path] += 1
+
+        if len(paths_freq) == 0:
+            return None
+
+        conf_dir = max(paths_freq, key=(lambda k: paths_freq[k]))
```

Comment:
Perhaps at this point, if there is more than one path in use, and you're picking the most frequently used, add a `logger.info` to tell the user that is happening?  Otherwise this behaviour could be totally mysterious?